src/data/pokemon/form_change_tables.h

```diff
@@ -778,9 +778,9 @@ static const struct FormChange sDuraludonFormChangeTable[] =
     {FORM_CHANGE_TERMINATOR},
 };
 
-static const struct FormChange sUrshifuFormChangeTable[] =
+static const struct FormChange sUrshifuSingleStrikeFormChangeTable[] =
 {
-    {FORM_CHANGE_BATTLE_GIGANTAMAX, SPECIES_URSHIFU_GIGANTAMAX},
+    {FORM_CHANGE_BATTLE_GIGANTAMAX, URSHIFU_SINGLE_STRIKE_STYLE},
```

Comment:
I think this is (at least one of the places) where I messed up my merge conflicts?
```suggestion
    {FORM_CHANGE_BATTLE_GIGANTAMAX, SPECIES_URSHIFU_SINGLE_STRIKE_STYLE},
```

Reply:
This was it, already fixed it :)